src/figuresClasses.ts

```diff
@@ -1,19 +1,96 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: 'triangle' = 'triangle';
+
+  color: 'red' | 'green' | 'blue';
+
+  a: number;
+
+  b: number;
+
+  c: number;
 
+  constructor(color: 'red'
+  | 'green'
+  | 'blue',
+  a: number, b: number, c: number) {
```

Comment:
```suggestion
  constructor(
  public color: Color,
  public a: number,
  public b: number, 
  public c: number
  ) {
```

Reply:
Create types and use in all classes instead hardcoding same union types.
Use public keyword in all constructors